quantum/quantum.c

```diff
@@ -481,3 +481,40 @@ void api_send_unicode(uint32_t unicode) {
 __attribute__((weak)) void startup_user() {}
 
 __attribute__((weak)) void shutdown_user() {}
+
+
+/** \brief converts unsigned integers into char arrays
+ *
+ * Takes an unsigned integer, and uses a static conversion buffer to convert that value into a
+ * char array
```

Comment:
```suggestion
 * Takes an unsigned integer, and uses a static conversion buffer to convert that value into a
 * char array
 * NOTE: Subsequent invocations will reuse the same static buffer and overwrite the previous
 *       contents. Use the result immediately, instead of caching it.
```